src/change_trust_asset.js

```diff
@@ -0,0 +1,271 @@
+import clone from 'lodash/clone';
+import padEnd from 'lodash/padEnd';
+import trimEnd from 'lodash/trimEnd';
+import xdr from './generated/stellar-xdr_generated';
+import { Asset } from './asset';
+import { Keypair } from './keypair';
+import { LiquidityPoolFeeV18, getLiquidityPoolId } from './liquidity_pool_id';
+import { StrKey } from './strkey';
+
+/**
+ * ChangeTrustAsset class represents a trustline change to either a liquidity
+ * pool, the native asset (`XLM`) or an issued asset with an asset code / issuer
+ * account ID pair.
+ *
+ * In case of the native asset, the code will represent `XLM` while the issuer
+ * and liquidityPoolParameters will be empty. For an issued asset, the code and
+ * issuer will be valid and liquidityPoolParameters will be empty. For liquidity
+ * pools, the liquidityPoolParameters will be valid and the remaining two fields
+ * will be empty.
+ *
+ * @constructor
+ * @param {string} code - The asset code.
+ * @param {string} issuer - The account ID of the asset issuer.
+ * @param {LiquidityPoolParameters} liquidityPoolParameters – The liquidity pool parameters.
+ * @param {Asset} liquidityPoolParameters.assetA – The first asset in the Pool, it must respect the rule assetA < assetB.
+ * @param {Asset} liquidityPoolParameters.assetB – The second asset in the Pool, it must respect the rule assetA < assetB.
```

Comment:
Can we drop a reference to something explaining what `<` means here? I see `validateLexicographicAssetsOrder()` which I think is relevant.

Reply:
Do you think this works? https://github.com/stellar/js-stellar-base/pull/454/commits/35771f2bb76ef89e08100bcab55fadde27c544f8

I didn't want to duplicate the information here so I just referenced `validateLexicographicAssetsOrder `.

Reply:
Yeah something like that is all I was looking for, or even a `@see validateLexicographicAssetsOrder`; thanks!